mlrun/datastore/__init__.py

```diff
@@ -123,7 +123,7 @@ def get_stream_pusher(stream_path: str, **kwargs):
         )
         if isinstance(
             datastore_profile,
-            (DatastoreProfileKafkaSource, DatastoreProfileKafkaTarget),
+            (DatastoreProfileKafkaStream, DatastoreProfileKafkaTarget),
```

Comment:
I think here and in below you cannot discard DatastoreProfileKafkaSource for backward compatability sake. Imagine you have datastore profile already define in 1.9 - you will have object of type 'kafka_source' kept in the database. So you need to keep all the functionality for DatastoreProfileKafkaSource